core/src/unprocessed_packet_batches.rs

```diff
@@ -155,13 +159,14 @@ impl PartialOrd for ImmutableDeserializedPacket {
 
 impl Ord for ImmutableDeserializedPacket {
     fn cmp(&self, other: &Self) -> Ordering {
-        match self.fee_per_cu().cmp(&other.fee_per_cu()) {
+        match self.fee_per_cu().partial_cmp(&other.fee_per_cu()).unwrap_or(Ordering::Equal) {
```

Comment:
seems like these comparisons should never return `None`: https://doc.rust-lang.org/src/core/cmp.rs.html#1396-1398, so we can unwrap on all these

Reply:
ah, done